common/app/model/dotcomrendering/DotcomRenderingUtils.scala

```diff
@@ -503,8 +503,8 @@ object DotcomRenderingUtils {
       tags = allTags, // List[Tag]
       pillar = findPillar(article.metadata.pillar, article.metadata.designType), // String
       isImmersive = article.isImmersive, // Boolean
-      sectionLabel = Localisation(article.content.sectionLabelName)(request), // String
-      sectionUrl = article.content.sectionLabelLink, // String
+      sectionLabel = Localisation(article.content.sectionLabelName.getOrElse(""))(request), // String
+      sectionUrl = article.content.sectionLabelLink.getOrElse(""), // String
```

Comment:
Could we remove as well all the comments adding type?

Reply:
Just added a commit who do such a thing

Reply:
Thanks @mchv. I did that the other day to make a colleague happy, but yeah, it's cleaner now :)

Reply:
Added a bit of reasoning on why you don't need those comments:
- The expected by the left side will be immediately available in your IDE or text editor through jumping to the definition
- Even if you use a simple text editor, I do, the compiler will ensure you use the right type and break if you do not
- Even if you like editing directly in `github`, I do, having the type is either obvious (`getOrElse("")`) or not so helpful because the opacity of the right side.

Reply:
Agreed. All things considered, I am glad you removed them 🙂